common/static/common/templates/discussion/thread-response-show.underscore

```diff
@@ -1,4 +1,7 @@
-<header>
+<header class="wrapper-response-header">
+  <div class="author-image level-response">
+    <img class="image-frame" src="https://d1kekzok76m982.cloudfront.net/prod/5d0da5229e4c83894a51e04a41e06cd0_120.jpg?v=1441780917" alt="Profile image for post author">
```

Comment:
Will the "Profile image for post author" ``alt`` value show the real user's username at runtime?